api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirMedicationDaoImpl.java

```diff
@@ -28,16 +27,18 @@
 @Setter(AccessLevel.PACKAGE)
 public class FhirMedicationDaoImpl extends BaseFhirDao<Drug> implements FhirMedicationDao {
 	
+	/*
```

Comment:
This was here because I believe we had errors when creating a drug that the ingredients weren't being saved. If that's no longer the case, we can just remove this method altogether.